<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-20</y>
     <width>1281</width>
     <height>851</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.426, y1:0.00568182, x2:0.517, y2:1, stop:0 rgba(100, 241, 255, 255), stop:1 rgba(245, 221, 221, 255));</string>
   </property>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>130</y>
      <width>1301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>851</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.205, y1:0, x2:0.312, y2:1, stop:0 rgba(100, 241, 255, 0), stop:1 rgba(245, 221, 221, 0));
font: 36pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>How suitable are you for this job?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextquestion">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>440</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(179, 229, 255);</string>
    </property>
    <property name="text">
     <string>Next -&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>501</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="questions_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>911</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(179, 229, 255);</string>
    </property>
    <property name="text">
     <string>Question:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="points">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>570</y>
      <width>191</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(0, 0, 0);
background-color: rgb(179, 229, 255);</string>
    </property>
    <property name="text">
     <string>  0-20 = No
  21 - 40 = I'm not sure
  41 - 60 = Maybe
  61 - 80 = I think yes
  81 - 100 = Absolutely</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>570</y>
      <width>16</width>
      <height>241</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="done_btn">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>710</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(179, 229, 255);</string>
    </property>
    <property name="text">
     <string>I'm ready!!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>720</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(179, 229, 255);</string>
    </property>
    <property name="text">
     <string>Your points for now:</string>
    </property>
   </widget>
   <widget class="QLabel" name="points_show">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>730</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(179, 229, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="prevquestion">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(179, 229, 255);</string>
    </property>
    <property name="text">
     <string>&lt;- Previous</string>
    </property>
   </widget>
   <zorder>line_5</zorder>
   <zorder>line_2</zorder>
   <zorder>label</zorder>
   <zorder>nextquestion</zorder>
   <zorder>line</zorder>
   <zorder>line_3</zorder>
   <zorder>questions_label</zorder>
   <zorder>points</zorder>
   <zorder>label_2</zorder>
   <zorder>line_4</zorder>
   <zorder>done_btn</zorder>
   <zorder>label_3</zorder>
   <zorder>points_show</zorder>
   <zorder>prevquestion</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
